src/views/IdentityOverview/IdentityOverview.tsx

```diff
@@ -123,7 +123,11 @@ export function IdentityOverview({ identity }: Props): JSX.Element | null {
 
       {upgradeDid && (
         <Link
-          to={generatePath(paths.identity.did.upgrade.start, { address })}
+          to={
+            did
+              ? generatePath(paths.identity.did.upgrade.start, { address })
+              : paths.home
+          }
```

Comment:
Wouldn't it be better not to render the link at all if there's no DID? The home screen will only redirect back to the current identity overview screen.

Reply:
Yeah, I had to fix things